src/store/modules/TsModsModule.ts

```diff
@@ -118,6 +119,17 @@ export const TsModsModule = {
             return getters.cachedMod(mod).isLatest;
         },
 
+        /*** Was the last successful mod list update more than an hour ago? */
+        isModListOutdated(state) {
+            return state.modsLastUpdated instanceof Date
+                && (Date.now() - state.modsLastUpdated.getTime()) > (1000 * 60 * 60);
+        },
+
+        /*** Is the error from only some of the package chunks failing to be fetched/cached? */
+        isStatusPartiallyFailed(state) {
+            return state.thunderstoreModListUpdateError?.message === PARTIAL_UPDATE_ERROR;
```

Comment:
You can't use `?` like that in this project, as shown by the CI pipeline failure.